splunk_eventgen/lib/plugins/generator/jinja.py

```diff
@@ -195,8 +195,12 @@ def gen(self, count, earliest, latest, samplename=None):
                 app_name = getattr(self._sample, 'app', 'SA-Eventgen')
                 working_dir = os.path.join(splunk_home, 'etc', 'apps', app_name, 'default')
 
+            default_template_dir = os.path.join(os.path.dirname(working_dir), 'samples', 'templates')
+            if hasattr(self._sample, "sampleDir"):
```

Comment:
I have fixed #176 in my local branch and will submit the PR soon. 

I think this only fixed part of the issue. for #176, we need to fix 2 parts
1. resolve the jinja_template_dir with sampleDir when it is a relative path
2. use the jinja_template_dir directly when it is a absolute path.

Reply:
This issue is addressed in https://github.com/splunk/eventgen/pull/177

Please review this.

Reply:
@jmeixensperger could we just use #177 since it addressed both the `default template dir` issue and `vulnerability ` issue?